api/db/run-migrations.sh

```diff
@@ -2,15 +2,9 @@
 set -xeuo pipefail
 IFS=$'\n\t'
 
-activity="-PrunList=$1"
-if [ -z ${2+x} ]
-then
-    context=""
-else
-    context="-Pcontexts=$2"
-fi
-
-# Ruby is not installed in our dev container and this script is short, so bash is fine.
+liquibaseCommand="${1-update}"
+runList="-PrunList=${2-main}"
+liquibaseArgs="${3--PliquibaseCommandValue=}"
```

Comment:
This is really gross, but it works. I.e. we are handing the liquibase plugin an empty command value, and it doesn't break anything.

This script is way past my Bash expertise/tolerance. If I need to keep it, I want to port to Ruby or Python, or even pure Gradle.